pkg/log/log.go

```diff
@@ -93,10 +92,6 @@ func initEnvVarFields() {
 			envVarFields = field.Add(envVarFields, strings.ToLower(e), ev)
 		}
 	}
-
-	if clsName, err := config.GetClusterName(nil); err == nil {
-		envVarFields = field.Add(envVarFields, "cluster_name", clsName)
-	}
```

Comment:
Is it possible to remove init() only from kando command execution path? Can we keep that in the controller path?

Reply:
I think what we need to decide is, is it ok to remove cluster_name from log fields or not. And then we can decide on the solution.

Reply:
@pavannd1, @hairyhum what do you think?

Reply:
I also think removing information from the logs where it's needed is not a good thing. If we can keep cluster_name for controller logs - that would be better.